sdk/containerregistry/azure-containerregistry/tests/testcase.py

```diff
@@ -186,6 +179,9 @@ def load_registry():
     if not is_live():
         return
     authority = get_authority(os.environ.get("CONTAINERREGISTRY_ENDPOINT"))
+    authority_anon = get_authority(os.environ.get("CONTAINERREGISTRY_ANONREGISTRY_ENDPOINT"))
+    registry_name = os.environ["CONTAINERREGISTRY_REGISTRY_NAME"]
```

Comment:
nit: consider using `os.environ.get()` instead of `os.environ[]` here (and on the following line) to align with the lines above. That would also make it possible to run non-`anon` tests without setting the anonymous registry variables

Reply:
Thanks, have updated!